scrape/scrape.go

```diff
@@ -191,6 +191,12 @@ var (
 		},
 		[]string{"scrape_job"},
 	)
+	targetScrapeNativeHistogramBucketLimit = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "prometheus_target_scrapes_histogram_exceeded_bucket_limit_total",
```

Comment:
Maybe, for consistency with the config option and how the sample limit metric is called, call this `prometheus_target_scrapes_exceeded_bucket_limit_total`, or, if we change the config option as suggested above, `prometheus_target_scrapes_exceeded_histogram_bucket_limit_total` or `prometheus_target_scrapes_exceeded_native_histogram_bucket_limit_total`.